client/app/react-components/QueryMetadata.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import { scheduleHumanize } from '@/filters/index';
+import ScheduleDialog from './ScheduleDialog';
+
+function timeAgo(when) {
+  return when ? moment(when).fromNow() : '-';
+}
+
+export default class QueryMetadata extends React.Component {
+  static propTypes = {
+    mobile: PropTypes.bool.isRequired,
+    query: PropTypes.object.isRequired,
+    saveQuery: PropTypes.func.isRequired,
+    canEdit: PropTypes.bool.isRequired,
+    canScheduleQuery: PropTypes.bool.isRequired,
+  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      showScheduleDialog: false,
+    };
+  }
+
+  openScheduleForm = () => {
+    if (!(this.props.canEdit || this.props.canScheduleQuery)) {
+      return;
+    }
+    this.setState({ showScheduleDialog: true });
+  }
+
+  closeScheduleForm = () => this.setState({ showScheduleDialog: false });
+
+  render() {
+    if (this.props.mobile) {
+      return (
+        <div className="row query-metadata__mobile">
+          <div className="col-xs-4 text-left">
+            <span className="m-r-5">Created by</span>
+            <img alt="" src={this.props.query.user.profile_image_url} className="profile__image_thumb" /> <strong>{timeAgo(this.props.query.created_at)}</strong>
+          </div>
+          {!this.props.query.id ? '' :
+          <div className="col-xs-4 text-center">
+            <span className="m-r-5">Updated by</span>
+            <img alt={this.props.query.user.name} src={this.props.query.last_modified_by.profile_image_url} className="profile__image_thumb" /><strong>{timeAgo(this.props.query.updated_at)}</strong>
+          </div> }
+          <div className="col-xs-4 text-right">
+            <ScheduleDialog
+              show={this.state.showScheduleDialog}
+              query={this.props.query}
+              saveQuery={this.props.saveQuery}
+              refreshOptions={this.props.clientConfig.queryRefreshIntervals}
+              onClose={this.closeScheduleForm}
+            />
+            <span className="query-metadata__property">Refresh Schedule</span>
+            {!this.props.query.id ?
+              <span>Never</span> :
+              <a role="button" tabIndex="0" onKeyPress={this.openScheduleForm} onClick={this.openScheduleForm}>{scheduleHumanize(this.props.schedule)}</a>}
+          </div>
+        </div>
+      );
+    }
+    if (!this.props.query.id) {
+      return null;
+    }
+    return (
+      <div className="query-metadata query-metadata--history">
+        <table>
+          <tbody>
+            <tr>
+              <td>
+                <img alt={this.props.query.user.name} src={this.props.query.user.profile_image_url} className="profile__image_thumb" /> <strong className="meta__name">{this.props.query.user.name}</strong>
+              </td>
+              <td className="text-right">
+                created <strong>{timeAgo(this.props.query.created_at)}</strong>
```

Comment:
It's up for interpretation, but I'm not sure if this is an appropriate use of `strong`. From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong):

> The &lt;strong&gt; element is for content that is of "strong importance," including things of great seriousness or urgency (such as warnings). This could be a sentence that is of great importance to the whole page, or you could merely try to point out that some words are of greater importance compared to nearby content... However, it should not be used simply to apply bold styling; use the CSS font-weight property for that purpose.

Also, whether or not you use `<strong>` I'd consider using [`<time>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time).